### PR TITLE
Refactor EC2 and ELB exposed ports CloudFormation queries for performance gains 

### LINKED ISSUE #1718: Refactor EC2 and ELB exposed ports CloudFormation queries for performance gains
**Platform**
CloudFormation

**Description**
The load balancer or instance has a sensitive port connection is exposed to the entire internet

### PR DESCRIPTION
**Pull request by rogeriopeixotocx**
**Platform**
CloudFormation

**Description**
- The load balancer has a sensitive port connection is exposed to the entire internet
- The instance has a sensitive port connection is exposed to the entire internet
- Adding benchmark tests

Closes #1718

### CODE REVIEW

**Comment by rogeriopeixotocx** [comment]
@ruigomescx please don't merge yet, I want to compare the performance with the previous version.

File: assets/queries/cloudFormation/lb_sensitive_tcp_port_is_exposed_to_entire_network/query.rego [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                       contains(loadBalancer.Type, "ElasticLoadBalancing")]
+
+  elb := loadBalancerList[j]
+  elbType := getELBType(elb.properties)
+
+  securityGroupList = [{"name": key, "properties": secGroup}|
+                       secGroup := resources[key];
+                       contains(secGroup.Type, "SecurityGroup")]
+
+  secGroup := securityGroupList[k]
```
Comment (joaomartinscx) [inline assets/queries/cloudFormation/lb_sensitive_tcp_port_is_exposed_to_entire_network/query.rego:81]: How is elbType or elb related to secGroup? At first sight, doesn't seem to interfere with the following checks in ingress ports in security groups, unless the goal is just to verify its existence before doing the next verifications.

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

File: assets/queries/cloudFormation/lb_sensitive_tcp_port_is_exposed_to_entire_network/query.rego (hunk 1 above)
Comment (rogeriopeixotocx) [reply assets/queries/cloudFormation/lb_sensitive_tcp_port_is_exposed_to_entire_network/query.rego]: solved!

File: assets/queries/cloudFormation/elb_sensitive_port_is_exposed_to_entire_network/metadata.json [hunk 2]
```diff
@@ -1,8 +1,8 @@
 {
-  "id": "lb_sensitive_port_is_exposed_to_entire_network",
-  "queryName": "LB Sensitive Port Is Exposed To Entire Network",
+  "id": "elb_sensitive_tcp_port_is_exposed_to_entire_network",
+  "queryName": "ELB Sensitive TCP Port Is Exposed To Entire Network",
```
Comment (joaomartinscx) [inline assets/queries/cloudFormation/elb_sensitive_port_is_exposed_to_entire_network/metadata.json:3]: This shouldn't be generic to all ports?

**Review by joaomartinscx — CHANGES_REQUESTED** [review]

File: assets/queries/cloudFormation/elb_sensitive_port_is_exposed_to_entire_network/metadata.json (hunk 2 above)
Comment (rogeriopeixotocx) [reply assets/queries/cloudFormation/elb_sensitive_port_is_exposed_to_entire_network/metadata.json:3]: yep!

**Review by ruigomescx — APPROVED** [review]

**Review by ruigomescx — APPROVED** [review]

**Pull request merged**